openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/superset.spec.js

```diff
@@ -20,7 +20,7 @@ import {
     updateDescriptionForIngestedTables,
     uuid
 } from '../../common/common';
-import { LOGIN, SERVICE_TYPE } from '../../constants/constants';
+import { DELETE_SERVICE, LOGIN, SERVICE_TYPE } from '../../constants/constants';
```

Comment:
instead of giving a particular name as `DELETE_SERVICE` give a generic name like `SERVICE_TYPE`

Reply:
Changed it to ```API_SERVICE```